Specs/DateTime/Portuguese/DateTimeModel.json

```diff
@@ -3120,5 +3120,535 @@
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "a palestra começa às 8 horas am.",
+    "Context": {
+      "ReferenceDateTime": "2020-05-14T12:00:00"
+    },
+    "NotSupported": "javascript, java",
+    "Results": [
+      {
+        "Text": "8 horas am",
+        "Start": 21,
+        "End": 30,
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T08",
+              "type": "time",
+              "value": "08:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "vou voltar em 14 de outubro 8 horas am",
+    "Context": {
+      "ReferenceDateTime": "2021-03-15T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "14 de outubro 8 horas am",
+        "Start": 14,
+        "End": 37,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-10-14T08",
+              "type": "datetime",
+              "value": "2020-10-14 08:00:00"
+            },
+            {
+              "timex": "XXXX-10-14T08",
+              "type": "datetime",
+              "value": "2021-10-14 08:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "vou voltar 19 horas 2016 - 12 - 22",
+    "Context": {
+      "ReferenceDateTime": "2021-03-15T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "19 horas 2016 - 12 - 22",
+        "Start": 11,
+        "End": 33,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2016-12-22T19",
+              "type": "datetime",
+              "value": "2016-12-22 19:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "vou voltar 8 horas am 2016 - 12 - 22",
+    "Context": {
+      "ReferenceDateTime": "2021-03-15T00:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "8 horas am 2016 - 12 - 22",
+        "Start": 11,
+        "End": 35,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2016-12-22T08",
+              "type": "datetime",
+              "value": "2016-12-22 08:00:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "estarei fora de 4 horas a 5 pm 5 de maio 2018",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T12:30:00"
+    },
+    "NotSupported": "javascript, java, python",
+    "Results": [
+      {
+        "Text": "de 4 horas a 5 pm 5 de maio 2018",
+        "Start": 13,
+        "End": 44,
+        "TypeName": "datetimeV2.datetimerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(2018-05-05T04,2018-05-05T17,PT13H)",
```

Comment:
The usual interpretation of this would be T16 as the end is T17.
Either that should be returned, or both interpretations for "de 4 horas"

Reply:
fixed